db/system_keyspace.cc

```diff
@@ -1724,6 +1724,10 @@ future<> system_keyspace::update_tokens(const std::unordered_set<dht::token>& to
 
 future<> system_keyspace::force_blocking_flush(sstring cfname) {
     assert(qctx);
+    if (!qctx->_qp.local().db().get_config().flush_schema_tables_after_modification()) {
```

Comment:
Pear, mark the method noon-static, you will be able to use system_keyspace::_db directly

Reply:
This part is now merged.